README.md

```diff
@@ -10,23 +10,23 @@ A collection of code examples which generate modelling features for a mobile pho
 
 | Name                                                                                                            | Description                                                                                                                 |
 | --------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| [average_end_of_day_depository_balance](lib/average_end_of_day_depository_balance)                              | average end-of-day total balance across all depository accounts                                                             |
```

Comment:
Might change to 'average end-of-day balance total across all depository accounts'

Reply:
Done. I also updated the one for the loans.